Resources/Prototypes/_Harmony/Entities/Objects/Weapons/Guns/Ammunition/Magazines/energycartridges.yml

```
@@ -99,3 +99,30 @@
   - type: Tag
     tags:
       - EnergyCarbineCartridge
+
+- type: entity
+  parent: [BaseSyndicateContraband, BaseEnergyCartridgeHarmony]
+  id: EnergyCarbineCartridgeSyndicateHarmony
+  name: jailbroken energy cartridge
+  description: A modified energy cartridge that is only compatible with the Blood-Red Energy Carbine. It shoots deadly shock projectiles.
+  components:
+  - type: Sprite
+    sprite: _Harmony/Objects/Weapons/Guns/Ammunition/Magazines/EnergyCartridges/Carbine/jailbroken.rsi
+    layers:
+    - state: base
+      map: ["enum.GunVisualLayers.Base"]
+    - state: mag-1
+      map: ["enum.GunVisualLayers.Mag"]
+      shader: unshaded
+  - type: MagazineVisuals
+    magState: mag
+    steps: 4
+  - type: Battery
+    maxCharge: 750
+    startingCharge: 750
+  - type: ProjectileBatteryAmmoProvider
+    proto: BulletLaserShockHarmony
+    fireCost: 37
+  - type: Tag
+    tags:
+      - EnergyCarbineCartridgeSyndicate
```

You're one indentation level too high, the dashes to define a list in yaml isn't considered the list itself and shouldn't be indented.
```suggestion
    tags:
    - EnergyCarbineCartridgeSyndicate
```